gemini/evaluation/evaluating_prompts_at_scale_with_gemini_batch_prediction_api.ipynb

```diff
@@ -0,0 +1,1011 @@
+    "        ],\n",
+    "        source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,\n",
+    "        write_disposition=bigquery.WriteDisposition.WRITE_TRUNCATE,\n",
+    "    )\n",
+    "\n",
+    "    with open(json_file_name, \"rb\") as source_file:\n",
+    "        job = bq_client.load_table_from_file(\n",
+    "            source_file, PROMPTS_TABLE, job_config=job_config\n",
+    "        )\n",
+    "\n",
+    "    job.result()\n",
+    "    table = bq_client.get_table(prompts_table)\n",
+    "    print(\n",
+    "        \"Loaded {} rows and {} columns to {}\".format(\n",
+    "            table.num_rows, len(table.schema), prompts_table\n",
+    "        )\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "upload_newline_json_file(\n",
+    "    json_file_name=json_file_name, project_id=PROJECT_ID, prompts_table=PROMPTS_TABLE\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LveMID56Ngo6"
+   },
+   "source": [
+    "### Launch a Gemini Batch Prediction request"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "-NdypZy-TacS",
+    "outputId": "a1df0622-287a-4e77-f280-d68afe94c843"
+   },
+   "outputs": [],
+   "source": [
+    "# Define table to store results from Gemini Batch Prediction.\n",
+    "text_generation_table = f\"{TEXT_GENERATION_TABLE_PREFIX}_{evaluation_id}\"\n",
+    "\n",
+    "\n",
+    "def create_batch_prediction_job(\n",
+    "    project_id, model_id, prompts_table, text_generation_table\n",
+    "):\n",
+    "    \"\"\"Creates a Gemini batch prediction job.\"\"\"\n",
+    "\n",
+    "    gemini_batch_url = (\n",
+    "        f\"https://us-central1-aiplatform.googleapis.com/v1/projects/{project_id}/\"\n",
+    "        f\"locations/us-central1/batchPredictionJobs\"\n",
+    "    )\n",
+    "\n",
+    "    # Get Authentication token.\n",
+    "    creds, _ = google.auth.default()\n",
+    "    auth_req = google.auth.transport.requests.Request()\n",
+    "    creds.refresh(auth_req)\n",
+    "    token = creds.token\n",
+    "\n",
+    "    # Build request.\n",
+    "    request_data = {\n",
+    "        \"displayName\": evaluation_id,\n",
+    "        \"model\": f\"publishers/google/models/{model_id}\",\n",
+    "        \"inputConfig\": {\n",
+    "            \"instancesFormat\": \"bigquery\",\n",
+    "            \"bigquerySource\": {\"inputUri\": f\"bq://{prompts_table}\"},\n",
+    "        },\n",
+    "        \"outputConfig\": {\n",
+    "            \"predictionsFormat\": \"bigquery\",\n",
+    "            \"bigqueryDestination\": {\"outputUri\": f\"bq://{text_generation_table}\"},\n",
+    "        },\n",
+    "    }\n",
+    "    headers = {\n",
+    "        \"Content-Type\": \"application/json\",\n",
+    "        \"Authorization\": f\"Bearer {token}\",\n",
+    "    }\n",
+    "\n",
```

Comment:
Can you change this to use the Vertex AI SDK?

It seems like this feature was added, but no official code samples have been created yet.

https://github.com/googleapis/python-aiplatform/blob/main/vertexai/batch_prediction/_batch_prediction.py#L240